### PR TITLE
Fix 8.1 integration tests

### PR DESCRIPTION
**Pull request by ZNeumann**
Passing tests dependent on [PR-324](https://github.com/newrelic/newrelic-php-agent/pull/324). That PR's code changes are included in this PR as well.

### CODE REVIEW

**Review by mfulb — COMMENTED** [review]
A few comments:
- Perhaps use a directory level skipif.inc for the guzzle5 case instead of adding it to each test case in the directory?
- Would it be appropriate to add test cases for PHP 8.1 and pgsql to use an object to mirror the cases which were skipif'd which used a resource?

**Comment by ZNeumann** [comment]
ok jenkins

**Comment by ZNeumann** [comment]
ok jenkins

**Comment by ZNeumann** [comment]
ok jenkins

**Comment by zsistla** [comment]
ok jenkins

File: tests/include/integration.php [hunk 1]
```diff
@@ -8,16 +8,31 @@
...
-  'DatastoreInstance.php',
-  'Metric.php',
-  'SlowSQL.php',
-  'Trace/Segment.php',
-  'Trace/FilterIterator.php',
-  'Trace/StringTable.php',
-  'Trace.php',
-  'Transaction.php',
-);
+if (version_compare(PHP_VERSION, "8.1", "<")) {
```
Comment (zsistla) [inline tests/include/integration.php:11]: Nice!

**Review by zsistla — APPROVED** [review]

**Review by mfulb — APPROVED** [review]
Nice work Zach!

**Pull request merged**